include/zephyr/drivers/i3c/ccc.h

```diff
@@ -952,47 +1070,197 @@ struct i3c_ccc_getcaps {
+ */
+#define I3C_CCC_GETCAPS_TESTPAT					0xA55AA55A
+
+/**
+ * @brief Get Controller Handoff Capabilities Byte 1 (GETCAPS) Format 2 -
+ *        Hot-Join Support.
+ */
+#define I3C_CCC_GETCAPS_CRCAPS1_HJ_SUPPORT			BIT(0)
+
+/**
+ * @brief Get Controller Handoff Capabilities Byte 1 (GETCAPS) Format 2 -
+ *        Group Management Support.
+ */
+#define I3C_CCC_GETCAPS_CRCAPS1_GRP_MANAGEMENT_SUPPORT		BIT(1)
+
+/**
+ * @brief Get Controller Handoff Capabilities Byte 1 (GETCAPS) Format 2 -
+ *        Multi-Lane Support.
+ */
+#define I3C_CCC_GETCAPS_CRCAPS1_ML_SUPPORT			BIT(2)
+
+/**
+ * @brief Get Controller Handoff Capabilities Byte 2 (GETCAPS) Format 2 -
+ *        In-Band Interrupt Support.
+ */
+#define I3C_CCC_GETCAPS_CRCAPS2_IBI_TIR_SUPPORT			BIT(0)
+
+/**
+ * @brief Get Controller Handoff Capabilities Byte 2 (GETCAPS) Format 2 -
+ *        Controller Pass-Back.
+ */
+#define I3C_CCC_GETCAPS_CRCAPS2_CONTROLLER_PASSBACK		BIT(1)
+
+/**
+ * @brief Get Controller Handoff Capabilities Byte 2 (GETCAPS) Format 2 -
+ *        Deep Sleep Capable.
+ */
+#define I3C_CCC_GETCAPS_CRCAPS2_DEEP_SLEEP_CAPABLE		BIT(2)
+
+/**
+ * @brief Get Controller Handoff Capabilities Byte 2 (GETCAPS) Format 2 -
+ *        Deep Sleep Capable.
+ */
+#define I3C_CCC_GETCAPS_CRCAPS2_DELAYED_CONTROLLER_HANDOFF	BIT(3)
+
+/** Get Capabilities (GETCAPS) - VTCAP1 - Virtual Target Type bit shift value. */
+#define I3C_CCC_GETCAPS_VTCAP1_VITRUAL_TARGET_TYPE_SHIFT	0
+
+/** Get Capabilities (GETCAPS) - VTCAP1 - Virtual Target Type bitmask. */
+#define I3C_CCC_GETCAPS_VTCAP1_VITRUAL_TARGET_TYPE_MASK		\
+	(0x07U << I3C_CCC_GETCAPS_VTCAP1_VITRUAL_TARGET_TYPE_SHIFT)
+
+/**
+ * @brief Get Capabilities (GETCAPS) - VTCAP1 - Virtual Target Type
+ *
+ * Obtain Virtual Target Type value from VTCAP1 value
+ * obtained via GETCAPS format 2 VTCAP def byte.
+ *
+ * @param vtcap1 VTCAP1 value.
+ */
+#define I3C_CCC_GETCAPS_VTCAP1_VITRUAL_TARGET_TYPE(vtcap1)	\
+	(((vtcap1) &						\
+	  I3C_CCC_GETCAPS_VTCAP1_VITRUAL_TARGET_TYPE_MASK)	\
+	 >> I3C_CCC_GETCAPS_VTCAP1_VITRUAL_TARGET_TYPE_SHIFT)
+
+/**
+ * @brief Get Virtual Target Capabilities Byte 1 (GETCAPS) Format 2 -
+ *        Side Effects.
+ */
+#define I3C_CCC_GETCAPS_VTCAP1_SIDE_EFFECTS			BIT(4)
+
+/**
+ * @brief Get Virtual Target Capabilities Byte 1 (GETCAPS) Format 2 -
+ *        Shared Peripheral Detect.
+ */
+#define I3C_CCC_GETCAPS_VTCAP1_SHARED_PERIPH_DETECT		BIT(5)
+
+/** Get Capabilities (GETCAPS) - VTCAP2 - Interrupt Requests bit shift value. */
+#define I3C_CCC_GETCAPS_VTCAP2_INTERRUPT_REQUESTS_SHIFT	0
+
+/** Get Capabilities (GETCAPS) - VTCAP2 - Interrupt Requests bitmask. */
+#define I3C_CCC_GETCAPS_VTCAP2_INTERRUPT_REQUESTS_MASK		\
+	(0x03U << I3C_CCC_GETCAPS_VTCAP2_INTERRUPT_REQUESTS_SHIFT)
+
+/**
+ * @brief Get Capabilities (GETCAPS) - VTCAP2 - Interrupt Requests
+ *
+ * Obtain Interrupt Requests value from VTCAP2 value
+ * obtained via GETCAPS format 2 VTCAP def byte.
+ *
+ * @param vtcap1 VTCAP2 value.
```

Comment:
`@param` needs to be `vtcap2` to match the name. Ditto for the one below.

Reply:
yeah.... I fixed that in an earlier commit, but it came back with a bad rebase :(